test/integration/relations.test.js

```diff
@@ -86,82 +83,53 @@ describe.skip('Relations', function () {
         });
     });
 
-    describe('Many-to-Many: Simple Cases', function () {
-        // Use the default file named after the describe block, would be many-to-many-simple-cases
-        beforeEach(utils.db.init());
-
-        afterEach(utils.db.reset());
-
-        it('can match array in (single value)', function (done) {
-            const queryJSON = {'tags.slug': {$in: ['animal']}};
-
-            // Use the queryJSON to build a query
-            const query = makeQuery(queryJSON);
-
-            // Check any intermediate values
-            console.log(query.toQuery());
-
-            // Perform the query against the DB
-            query.select()
-                .then((result) => {
-                    console.log(result);
-
-                    result.should.be.an.Array().with.lengthOf(3);
-
-                    // Check we get the right data
-                    // result.should.do.something;
-
-                    done();
-                })
-                .catch(done);
-        });
-
-        it('can match array in (multiple values)', function (done) {
-            const queryJSON = {'tags.id': {$in: [2, 3]}};
-
-            // Use the queryJSON to build a query
-            const query = makeQuery(queryJSON);
-
-            // Check any intermediate values
-            console.log('query', query.toQuery());
-
-            // Perform the query against the DB
-            query.select()
-                .then((result) => {
-                    console.log(result);
-
-                    result.should.be.an.Array().with.lengthOf(4);
-
-                    // Check we get the right data
-                    // result.should.do.something;
+    describe('Many-to-Many', function () {
+        before(utils.db.init('relations1'));
 
-                    done();
-                })
-                .catch(done);
-        });
+        describe('OR', function () {
+            it('tags.slug IN (animal)', function () {
+                const mongoJSON = {'tags.slug': {$in: ['animal']}};
+                const query = makeQuery(mongoJSON);
 
-        it('can match array in (multiple values with x, y and xy)', function (done) {
-            const queryJSON = {'tags.id': {$in: [1, 2]}};
+                return query
+                    .select()
+                    .then((result) => {
+                        result.should.be.an.Array().with.lengthOf(3);
```

Comment:
We could definitely do better assertions here, but we currently don't have access to the relations in the result.